modules/installation-osp-api-scaling.adoc

```diff
@@ -0,0 +1,83 @@
+// Module included in the following assemblies:
+//
+// * networking/openstack/load-balancing-openstack.adoc
+
+[id="installation-osp-api-scaling_{context}"]
+= Scaling clusters by using Octavia
+
+If your cluster does not use Kuryr, create an Octavia load balancer and then
```

Comment:
@luis5tb So:

"If you want to use multiple API load balancers, or if your cluster does not use Kuryr, create an Octavia load balancer and then"

?

Reply:
perfect! go for it!

Reply:
@luis5tb TY. 

@eurijon Could you give this a final look? The only substantive change from the last preview build from a QE perspective should be https://github.com/openshift/openshift-docs/pull/23467/commits/be49298733d65b4667e23051913356927a6c0d54

Reply:
Hi @maxwelldb, I am taking this and I am ok with the change.

/lgtm